.github/workflows/release.yml

```diff
@@ -306,7 +310,7 @@ jobs:
           echo "runtime_filename_${{matrix.network}}=$release_wasm_filename" >> $GITHUB_OUTPUT
       - name: Install srtool-cli
         run: |
-          cargo install --git https://github.com/chevdor/srtool-cli
+          rustup run stable cargo install --git https://github.com/chevdor/srtool-cli
```

Comment:
If the self-hosted runner is ubuntu 22.04 and has glibc ^2.3, then we can use a pre-built binary for srtool-cli/srtool, and not need to install the rust toolchain.

Reply:
@mattheworris I think this is just a temp step. We are trying to get docker-in-docker working so we can remove the need to have anything special on the self-hosted runner.

Reply:
That said, yes, it is fine to have both installed on the CI base image and fine to build srtool-cli with stable.